### PR TITLE
Scaffold out profile view

### PR DESCRIPTION
**Pull request by levithomason**
This is a stateless scaffolding of the profile view and route.  Will add other tiny PRs for adding the link accounts buttons and finally the state.

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
       <Menu>
-        <MenuItem>Dashboard</MenuItem>
-        <MenuItem>Assignments</MenuItem>
-        <MenuItem>My Account</MenuItem>
+        <Link to='/profile'>
+          <MenuItem>Profile</MenuItem>
+        </Link>
+        <Link to='/dashboard'>
+          <MenuItem>Dashboard</MenuItem>
+        </Link>
```
Comment (levithomason) [inline src/components/Header/Header.js:13]: image

File: src/routes/index.js [hunk 2]
```diff
@@ -15,6 +16,7 @@ export default (
     <Route path='/login' component={LoginView} onEnter={sendToProtectedRoute} />
     <Route onEnter={requireAuth} component={AppLayout}>
       <Route path='/dashboard' component={Dashboard} />
+      <Route path='/profile' component={ProfileView} />
```
Comment (levithomason) [inline src/routes/index.js:19]: add the route to the protected area of our router.

File: src/views/ProfileView/ProfileView.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      <Grid className='equal width'>
+        <Column>
+          <Header.H1>Profile</Header.H1>
+        </Column>
+      </Grid>
+    )
+  }
+}
+
+export default ProfileView
```
Comment (levithomason) [inline src/views/ProfileView/ProfileView.js:20]: Dummy stateless view, for now.

image

**Comment by ckcollab** [comment]
Cool! Seems good, I would merge it but I'm not sure if you need to tweak anything?

**Pull request merged**